libs/langchain-cloudflare/src/embeddings.ts

```diff
@@ -1,4 +1,4 @@
-import { Ai } from "@cloudflare/ai";
+import { Ai } from "@cloudflare/ai/dist/ai.js";
```

Comment:
Por que?

Reply:
I was unable to run in `examples`, or in a local project due to this error:
```
file:///Users/bracesproul/code/lang-chain-ai/wt/brace/cloudflare-tools/libs/langchain-cloudflare/dist/embeddings.js:2
import { Ai } from "@cloudflare/ai";
         ^^
SyntaxError: The requested module '@cloudflare/ai' does not provide an export named 'Ai'
```
When I switched the import path the error went away.

Reply:
awe shit, in triggering this error again to show you, it's now persisting even with that new path. Will try and fix

Reply:
That package is on the way out! 

There is a [TypeScript SDK](https://github.com/cloudflare/cloudflare-typescript) now that wraps nicely...but also it's on the Binding `env.AI.run` not sure how to do this.

Reply:
I have a PR up that uses the native binding: https://github.com/langchain-ai/langchainjs/pull/5287